FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_multitask/DemoTasks/MultitaskMQTTExample.c

```diff
@@ -342,11 +369,32 @@ typedef struct publishElement
  * @param[in] pxMQTTContext MQTT context pointer.
  * @param[in] xNetworkContext Network context.
  * @param[in] xCleanSession If a clean session should be established.
+ *
+ * @return `MQTTSuccess` if connection succeeds, else appropriate error code
+ * from MQTT_Connect.
  */
 static MQTTStatus_t prvMQTTConnect( MQTTContext_t * pxMQTTContext,
                                     NetworkContext_t * pxNetworkContext,
                                     bool xCleanSession );
 
+/**
+ * @brief Form a TCP connection to a server.
+ *
+ * @param[in] pxNetworkContext Network context.
+ *
+ * @return `true` if connection succeeds, else `false`.
```

Comment:
I feel like should we be returning `pdPASS` and `pdFAIL` instead of true and false because prvDisconnectNetwork and prvConnectNetwork are not really predicates.

Reply:
Sure, I changed it